packages/design-system/src/components/Tooltip/Tooltip.tsx

```diff
@@ -95,19 +95,21 @@ export const Tooltip = ({ id, children, title, placement = 'top', ...rest }: Too
 				},
 				floating.refs.setReference,
 			)}
-			<FloatingPortal>
-				<div
-					{...getFloatingProps()}
-					id={safeId}
-					ref={floating.refs.setFloating}
-					className={styles.container}
-					style={{ display: isOpen ? 'block' : 'none', ...floating.floatingStyles }}
-					{...rest}
-				>
-					<FloatingArrow ref={arrowRef} context={floating.context} />
-					{title}
-				</div>
-			</FloatingPortal>
+			{!!title && (
```

Comment:
If no title, you will have a prop `'aria-describedby': safeId` on the element, but no tooltip.
Not great for accessibility.